.gitignore

```diff
@@ -6,3 +6,9 @@
 
 # Don't ignore the LICENSE file
 !LICENSE
+
```

Comment:
Ah, OK - wrt my previous comment then - why not merge these two patches - no point excluding everything and then immediately adding files back in.
And this commit doesn't just modify the .gitignore, it adds a bunch of .xml files as well (not noted in the commit message)